### DISCUSSION TITLE
AlienFlowSpace DAO (DeFi, DeSci, GameFi, ReFi, RWA) DAPP Wb3

### DISCUSSION THREAD

**Discussion by Alien69Flow** [community/community#136711] [Programming Help]
### Select Topic Area

Show & Tell

### Body

How do I redo repositories of a web 3 dapp like mine? AlienFlow.Space I did this in wordpress, this was done in AlienFlow.vercel.app but it left us stranded without access to repositories and now how do I redo them again directly here? react, next, node? I need to clarify and add everything in its place where it corresponds so that it looks as it should, how do I put everything in its place like links, images, logos, menus, pages, texts? Please, Thanks

**Comment by ebndev** [comment]
Hi @AitorAilen, thanks for posting in GitHub Discussions! 

I've gone ahead and moved this post to our `Programming Help` category, as this topic may be more relevant there and might help someone give you a nudge in the right direction. 😁

In the meantime, can you elaborate on this part:
> how do I put everything in its place like links, images, logos, menus, pages, texts?

Are you asking how you would code that?

**Reply by Alien69Flow** [reply]
claro como hago para que se vea como mi web de wordpress con todas las enlaces, imagenes, logos, menus, paginas, textos , colores, fuentes!? Gracias

**Comment by Alien69Flow** [comment]
Tengo que rehacer mi DAPP web 3 y prescindir de WordPress antes del día 10
de septiembre, rehacerla directamente en react, next, node... Por favor es
urgente , Gracias

**Comment by samus-aran** [comment]
Hi @AitorAilen ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking in more detail. Start off with the first step. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Your (many) other posts were removed because it contained content that violates our Community Code of Conduct, including posting the same comment in multiple places. As a result, we have hidden the duplicate comments, and we ask that you no longer cross-post.

For additional guidance, please review our [Community on Discussions Code of Conduct](https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md) and [GitHub’s Terms of Service](https://docs.github.com/en/github/site-policy/github-terms-of-service).

Best of luck!

**Reply by Alien69Flow** [reply]
Gracias entonces como puede rehacer aquí en GitHub directamente con Next.js, node.js, react.js la DAPP web 3?

**Comment by Alien69Flow** [comment]
Como hago entonces para crear un proyecto y respositorio para está dapp web
3 en GitHub? Veo en otros proyectos carpetas y demás archivos en el
repositorio cada quien pone unos nombres , estructura y organización,
porque dentro está a código javascript y Typescript principalmente con
react, next, node por lo que veo y hasta en la web 3, trabajaba
anteriormente con WordPress y unas probatinas en figma, vercel, webflow,
wix .... Aquí como hago para estructurar y organizar el respositorio puedo
ponerle los nombres que quiera o tengo que seguir unas pautas o algo de
estos códigos y lenguajes? Por favor, Gracias

**Comment by Alien69Flow** [comment]
Entonces como hago para trabajar en GitHub porque hay un montón de archivos, carpetas, ficheros y lineas de código y nadie está explicando nada!?

**Comment by Alien69Flow** [comment]
que coño le pasa al error, fallo, vulnerabiliad todo el puto rato!?
